server/DataAccessObjects/DriverDao.js

```diff
@@ -2,16 +2,80 @@ class DriverDao {
 
     constructor(gsDB) {
         this.gsDB = gsDB;
+        //this.driverDatabase = this.gsDB.collection("Drivers");
     }
 
     notifyAllValidDrivers(activeOrder) {
-        let driverDatabase = this.gsDB.collection("Drivers");
-        driverDatabase.where('capacity', '>=', activeOrder.getTotalQuantity()).get().then(snapshot => {
+        this.driverDatabase.where('capacity', '>=', activeOrder.getTotalQuantity()).get().then(snapshot => {
```

Comment:
was it not working before without the this?

Reply:
> was it not working before without the this?

This wasn't this week's ticket and I'm not sure what you mean by 'not working'